src/Application/Actions/Donations/Create.php

```diff
@@ -150,20 +147,11 @@ protected function action(Request $request, Response $response, array $args): Re
                 $donation->setCampaign($campaign);
             }
 
-            if ($forNewPaymentElement) {
-                // We don't need to set specific payment methods here, they are set on the client side by the payment
-                // element.
-                $stripeMethodProperties = [
-                    'automatic_payment_methods' => [
-                    'enabled' => true,
-                    'allow_redirects' => 'always',
-                    ]
-                ];
-            } else {
-                $stripeMethodProperties = $donation->getStripeMethodProperties();
-            }
             $createPayload = [
-                ...$stripeMethodProperties,
+                'automatic_payment_methods' => [
+                    'enabled' => true,
```

Comment:
The new payment option doesn't give us an option to do anything other than have automatic methods on the frontend, and payment confirmation will then fail if we use it and we don't have automatic methods set in the intent created here. See https://stripe.com/docs/js/elements_object/create_payment_element or it's maybe clearer in the typescript: https://github.com/stripe/stripe-js/blob/cac790b90d6448dccc61fb9ba5299b48fcad4409/types/stripe-js/elements/payment.d.ts#L199